classes/publication/Collector.php

```diff
@@ -115,6 +140,10 @@ public function getQueryBuilder(): Builder
         $qb = DB::table('publications as p')
             ->select(['p.*']);
 
+        if ($this->isScheduled) {
+            $qb->where('p.status', PKPSubmission::STATUS_SCHEDULED);
```

Comment:
Do we need this filter? For this particular task we can look at the `status` property of the `publication`

And to double check, do we need only publication with the scheduled status but not published?

Reply:
>And to double check, do we need only publication with the scheduled status but not published?

That was the request [here](https://github.com/pkp/pkp-lib/issues/11025#issuecomment-2729888258), on Option#2 which is the implementation I went with.